package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "compile": "tsc index.ts test/spec.ts --module commonjs --noImplicitAny",
     "coverage": "cat ./coverage/lcov.info | coveralls",
-    "test": "istanbul cover _mocha",
+    "test": "istanbul cover node_modules/mocha/bin/_mocha",
```

Comment:
Ah, that's necessary to run tests on Windows :( . Without it npm attempts to interpret `_mocha.cmd` as a JavaScript file and it fails. I've already linked the corresponding bug in istanbul: https://github.com/gotwarlost/istanbul/issues/90

It seems they are unwilling to fix it, so we have to live with it. But I can remove this change if you absolutely disagree with it.